elasticsearch/query.go

```diff
@@ -60,15 +60,20 @@ func (m MatchQuery) MarshalJSON() ([]byte, error) {
 
 // MultiMatchQuery provides a query that matches on a number of fields.
 type MultiMatchQuery struct {
-	Query  string
-	Fields []string
+	Query              string
+	Fields             []string
+	MinimumShouldMatch string
```

Comment:
It'd be nice to know what this means (either with a doc comment
or a link to the ES docs)